docs/Data Lake.md

```diff
@@ -154,6 +155,85 @@ Finally, on clicking the *Execute Rules* button, you will be able to get a preli
 
 ![Preliminary hitlist](../img/Discover/image13.png)
 
+##DepMap CCLE
+
+###Opening the app
+
+Upon opening the Discover portal on Polly, choose the Depmap data repository. The page should look something like this.
+
+![Selecting app](../img/DepMapCCLE/image1.png)
+
+After selecting the Depmap repository, you’ll be able to see a dashboard and the DepMap CCLE app icon. The dashboard will give you an overview of the repository, for instance, the information of different cell lines in the  repository, distribution of tissue lineages, metastatic status of samples, etc. among the datasets.
+
+![Kibana dashboard](../img/DepMapCCLE/image2.png)
+
+Click on the below icon to start the DepMap CCLE app.
+
+![DepMap icon](../img/DepMapCCLE/image3.png)
+
+The app will open and you should see the home page which contains two examples describing in brief  usage of the application.
```

Comment:
Once inside the application, you can find two examples on the home page that briefly describes the usage of the application

Reply:
changed